### PR TITLE
Dmirano/4265 subform button accessibility

### LINKED ISSUE #4265: [Accessibility] Hiding the add button when a subform is already open
Describe the task here and link or add dependencies to any applicable issues...

To prevent data loss, we disabled the ability to have multiple subforms open at once. We solved this issue by hiding the add button if a subform is already open. WAVE caught this functionality and marked it as an error because we are hiding the button, potentially adding issues to users that rely on screen readers. We should come up with another way to prevent multiple subforms being open, but without hiding the add button.

Potential solution. Instead of hiding the button, we can leave the button in and gray it out if a subform is already open. We could also add a tooltip on hover that states something along the lines of "Please finish the current form before adding a new X"

### This task is done when...

- [ ] WAVE finds no errors with this issue

### PR DESCRIPTION
**Pull request by mirano-darren**
Resolves #4265 

### Description
Resolves a small accessibility issue for subforums when hiding the add button when a subforum is already open

### Significant changes or possible side effects
None, just added aria-labels 

### Automated test cases written
None

### Steps to manually verify this change
This was only found by the WAVE extension, so you would have to get that if you'd like to verify this change. There is a chrome extension here: https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US
(unsure about alternatives)

1. On staging, go to a page with a subforum and open it. The add button should disappear
2. Run WAVE on the page and verify it finds the "empty button" accessibility errors
3. Repeat with the preview build of this branch and verify WAVE finds no errors 

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] Associated OpenAPI documentation has been updated
- [x] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by thetif — APPROVED** [review]

File: web/src/pages/apd/activities/contractor-costs/ContractorResource/ContractorResourceForm.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ const ContractorResourceForm = forwardRef(
           disabled
           style={{ display: 'none' }}
           aria-hidden="true"
+          aria-label="submitButton"
```
Comment (tbolt) [inline web/src/pages/apd/activities/contractor-costs/ContractorResource/ContractorResourceForm.js:93]: Are these labels read to screen readers?

**Review by tbolt — APPROVED** [review]

File: web/src/pages/apd/activities/contractor-costs/ContractorResource/ContractorResourceForm.js (hunk 1 above)
Comment (mirano-darren) [reply web/src/pages/apd/activities/contractor-costs/ContractorResource/ContractorResourceForm.js:93]: Just tested. No they aren't read out loud

**Review by jeromeleecms — APPROVED** [review]
@mirano-darren Can you post a screenshot of the wave report? I can't test it anymore - apparently they disabled wave on my laptop.

If it passed the test, this is good to go.

**Comment by mirano-darren** [comment]
sure! Here's a screenshot @jeromeleecms 
Left one is this PRs preview build, and the right one is staging
image

**Pull request merged**